pkg/skaffold/constants/constants.go

```diff
@@ -60,6 +61,18 @@ const (
 	DefaultCloudBuildDockerImage = "gcr.io/cloud-builders/docker"
 	DefaultCloudBuildMavenImage  = "gcr.io/cloud-builders/mvn"
 	DefaultCloudBuildGradleImage = "gcr.io/cloud-builders/gradle"
+
+	// A regex matching valid repository names (https://github.com/docker/distribution/blob/master/reference/reference.go)
+	RepositoryComponentRegex string = `^[a-z\d]+(?:(?:[_.]|__|-+)[a-z\d]+)*$`
+
+	SkaffoldPluginKey       = "SKAFFOLD_PLUGIN_KEY"
+	SkaffoldPluginValue     = "42"
+	SkaffoldPluginName      = "SKAFFOLD_PLUGIN_NAME"
+	DockerBuilderPluginName = "docker"
+)
+
+var (
+	GoogleCloudBuild latest.ExecEnvironment = "googlecloudbuild"
```

Comment:
I'd personally prefer this to be `googleCloudBuild` since that's the convention we've been sticking to in the skaffold.yaml